README.md

```diff
@@ -47,11 +47,12 @@ It also comes with a web UI.
 
 ## Wiki
 
-For more information please check out the [GitHub wiki](https://github.com/pump-io/pump.io/wiki)
+For more information please check out the [GitHub wiki](https://github.com/pump-io/pump.io/wiki) and our [readthedocs](https://pumpio.readthedocs.io/en/latest/)
```

Comment:
`ReadTheDocs` instead of `readthedocs`, please. Also if you could add "documentation" at the end or something, so the sentence read:

> For more information...and our ReadTheDocs documentation.